webapp/src/main/java/com/box/l10n/mojito/service/pullrun/PullRunAssetService.java

```diff
@@ -77,7 +75,7 @@ void saveTextUnitVariantsMultiRowBatch(
       Long localeId,
       List<Long> uniqueTmTextUnitVariantIds,
       Instant now) {
-    String createdTime = Timestamp.valueOf(LocalDateTime.ofInstant(now, ZoneOffset.UTC)).toString();
+    String createdTime = Timestamp.from(now).toString();
```

Comment:
Should we be re-using UTC here as it was previously specified here?

Reply:
I can't seem to see this file in the PR anymore?

Reply:
yep, that was the only change and I changed it back as it was, so the file is no longer showing up as there is no diff.